packages/contracts/contracts/libs/v0.8.x/DependencyRegistryStorageLib.sol

```diff
@@ -66,9 +66,6 @@ library DependencyRegistryStorageLib {
         mapping(bytes32 dependencyNameAndVersion => Dependency dependency) dependencyRecords;
         // mapping from licenseTypes to License, which stores the properties of each license
         mapping(bytes32 licenseType => License license) allLicenses;
-        // Set of addresses for the core contracts that are supported by the DependencyRegistry.
-        // Each address represents a unique core contract in the Art Blocks ecosystem.
-        EnumerableSet.AddressSet supportedCoreContracts;
```

Comment:
cannot change existing storage layout in upgradeable contracts, ref https://docs.openzeppelin.com/upgrades-plugins/1.x/writing-upgradeable#modifying-your-contracts

Reply:
🙏 🙏  thanks for catching - I added this back with a dev comment